FrontDesk/src/FrontDesk.Api/appsettings.Docker.json

```diff
@@ -0,0 +1,18 @@
+﻿{
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=(localdb)\\mssqllocaldb;Integrated Security=true;Initial Catalog=FrontDesk.Web-20130819200722;",
+    "VetOfficeContext": "Server=(localdb)\\mssqllocaldb;Integrated Security=true;Initial Catalog=FrontDesk.Web-20130819200722;",
+    "ServiceBroker": "Server=(localdb)\\mssqllocaldb;Integrated Security=true;Initial Catalog=ServiceBrokerTest;"
```

Comment:
ServiceBroker can go away. Will be replaced by RabbitMQ or similar.
Probably only need one connection string, DefaultConnection. Anything using the other one should be updated to use DefaultConnection, I think.

Reply:
Done